fitter/probabilities/pulsars.py

```diff
@@ -128,46 +124,19 @@ def cluster_component(model, R, mass_bin, DM=None, ΔDM=None, DM_mdata=None, *,
     rhoz_spl = QuantitySpline(z, rho_spl(r), ext=1, s=0)
 
     # Now compute P(a_z|R)
-    # There are 2 possibilities depending on R:
-    #  (1) the maximum acceleration occurs within the cluster boundary, or
-    #  (2) max(a_z) = a_z,t (this happens when R ~ r_t)
 
-    nr, k = nz, 3  # bit of experimenting
+    nr = nz  # bit of experimenting
```

Comment:
I think the experimenting is done (though I wouldn't worry too much about the documentation right now, lot's of it needs to be redone)

Reply:
Yep, will update this too